File: Objetos/objeto_partida.js
//Objeto Partida com os atributos data, horario, duracao, jogador_brancas, jogador_pretas, jogadas_partida, vencedor
export class Partida{
    constructor(id_partida, data, horario, jogador_brancas, jogador_pretas, jogadas_partida, vencedor){
        this.id_partida = id_partida;
        this.data = data;
        this.horario = horario;
        this.jogador_brancas = jogador_brancas;
        this.jogador_pretas = jogador_pretas;
        this.jogadas_partida = jogadas_partida;
        this.vencedor = vencedor;
    }
    returnID_partida(){
        return this.id_partida;
    }

    returnData(){
        return this.data;
    }

    returnHorario(){
        return this.horario;
    }

    returnJogador_brancas(){
        return this.jogador_brancas
    }

    returnJogador_pretas(){
        return this.jogador_pretas;
    }

    returnJogadas_partida(){
        return this.jogadas_partida;
    }

    returnVencedor(){
        return this.vencedor;
    }
};

//lista de IDs e partidas
export let lista_ID_partidas = [];
export let lista_partidas = [];

File: Objetos/objeto_partida.py
#Objeto Partida com os atributos data, horario, duracao, jogador_brancas, jogador_pretas, jogadas_partida, vencedor
class Partida:
    def __init__(self, id_partida, data, horario, jogador_brancas, jogador_pretas, vencedor):
        self.id_partida = id_partida
        self.data = data
        self.horario = horario
        self.jogador_brancas = jogador_brancas
        self.jogador_pretas = jogador_pretas
        self.vencedor = vencedor
    
    def returnID_partida(self):
        return self.id_partida
    

    def returnData(self):
        return self.data
    

    def returnHorario(self):
        return self.horario
    

    def returnJogador_brancas(self):
        return self.jogador_brancas
    

    def returnJogador_pretas(self):
        return self.jogador_pretas
    

    def returnVencedor(self):
        return self.vencedor
    


#lista de IDs e partidas
lista_ID_partidas = []
lista_partidas = []